recipes/pyglpk/LICENSE.txt

```diff
@@ -0,0 +1,621 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I think you can remove the license file from the recipe because it is included in the PyPI source distribution.

Reply:
Committed those changes,  removing the license file and pointing it to the one in the source. Thanks!